File: bench/tasks/04-loops/go/main.go
package main

import "fmt"

func main() {
	fmt.Println("Countdown")
	remaining := 3
	for remaining > 0 {
		fmt.Println(remaining)
		remaining--
	}
	for i := 0; i < 2; i++ {
		fmt.Println("Go!")
	}
}

File: bench/tasks/04-loops/zig/main.zig
const std = @import("std");

pub fn main() !void {
    const stdout = std.io.getStdOut().writer();
    try stdout.print("Countdown\n", .{});
    var remaining: i64 = 3;
    while (remaining > 0) {
        try stdout.print("{}\n", .{remaining});
        remaining -= 1;
    }
    for (0..2) |_| {
        try stdout.print("Go!\n", .{});
    }
}
